lib/solidus_kustomer.rb

```diff
@@ -22,5 +22,27 @@ def configuration
     def configure
       yield configuration
     end
+
+    def create_customer_now(user)
+      kustomer_client.create_customer(user)
+    end
+
+    def create_customer_later(user)
+      SolidusKustomer::CreateCustomerJob.perform_later(user)
+    end
+
+    def update_customer_now(user)
+      kustomer_client.update_customer(user)
+    end
+
+    def update_customer_later(user)
+      SolidusKustomer::UpdateCustomerJob.perform_later(user)
+    end
```

Comment:
I see you're trying to replicate the API of solidus_tracking and solidus_klaviyo here. It may be useful, but if you add more methods to the client, you'd have to keep adding methods here, which is not ideal.

solidus_tracking and solidus_klaviyo have that API because they only expose a single piece of functionality and because those methods are expected to be called from the outside.

In your case, you expose multiple features, but these two methods in particular are already called by your callbacks, so I don't think you need to expose them like this. You can just call the client/background jobs directly where you need to use them.